.ci/pipeline_definitions

```diff
@@ -19,9 +19,9 @@ oidc-webhook-authenticator:
             tag_as_latest: false
     steps:
       check:
-        image: 'golang:1.17.9'
+        image: 'golang:1.18.2'
       test:
-        image: 'golang:1.17.9'
+        image: 'golang:1.18.2'
```

Comment:
This will have effect just after the PR is merged and the pipeline is re-rendered. So, it is expected the `test` check to fail.